src/jit/MemoryInl.h

```diff
@@ -515,7 +532,15 @@ static void emitLoadLaneSIMD(sljit_compiler* compiler, Instruction* instr)
         return;
     }
 
-    sljit_emit_simd_lane_mov(compiler, SLJIT_SIMD_LOAD | simdType, dstReg, loadOperation->index(), addr.memArg.arg, addr.memArg.argw);
+#if (defined SLJIT_CONFIG_ARM_32 && SLJIT_CONFIG_ARM_32)
+    if (simdType == (SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64)) {
+        sljit_emit_simd_mov(compiler, SLJIT_SIMD_LOAD | SLJIT_SIMD_REG_64 | SLJIT_SIMD_ELEM_8, loadOperation->index() ? getHighRegister(dstReg) : dstReg, addr.memArg.arg, addr.memArg.argw);
```

Comment:
`loadOperation->index() == 1` or put the `== 0` case before similar to `laneIndex == 0` below. I like the latter better